session_0/item.rb

```diff
@@ -1,45 +1,51 @@
 class Item
-  CATEGORIES = {
-    1 => 'Electronics/Gadgets',
-    2 => 'Books',
-    3 => 'Furniture',
-    4 => 'Miscellaneous'
+   CATEGORIES = {
+   1 => 'Electronics/Gadgets',
+   2 => 'Books',
+   3 => 'Furniture',
+   4 => 'Miscellaneous'
   }
-
-  # TODO: Store quantity
+  
   attr_accessor :name, :price, :category_id, :discount_rate,
-    :discount_deadline
-
-  def initialize(name: '', price: 0, quantity: 1, category_id: 4,
-                 discount_rate: 0, discount_deadline: Time.now)
-    @name = name
-    @price = price
-    @category_id = category_id
-    @discount_rate = discount_rate
-    @discount_deadline = discount_deadline
-    @quantity = quantity
-  end
-
-  # Returns a boolean value whether than item is discounted i.e. the
-  # discount deadline has been crossed or not.
+  :discount_deadline, :quantity
+  
+  def initialize(name: '', price:0, quantity:1, category_id:4, 
+                 discount_rate:0, discount_deadline: Time.now)
+   @name = name
+   @price = price
+   @category_id = category_id
+   @discount_rate = discount_rate
+   @discount_deadline = discount_deadline
+   @quantity = quantity
+  end 
+  
   def discounted?
-    raise NotImplementedError # TODO
+    if(Time.now>@discount_deadline)
```

Comment:
What happens when `Time.now` is equal to `@discount_deadline`? Is the item discounted or not?